src/MakingSense.AspNetCore.Authentication.SimpleToken/SimpleTokenAuthenticationHandler.cs

```diff
@@ -75,33 +84,39 @@ public static string ExtractToken(HttpRequest request)
 		/// <returns></returns>
 		protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
 		{
-			var token = ExtractToken(Request);
-
-			// If no token found, no further work possible
-			if (string.IsNullOrEmpty(token))
+			try
 			{
-				return AuthenticateResult.Skip();
-			}
+				var token = ExtractToken(Request);
 
-			var validationParameters = Options.TokenValidationParameters.Clone();
+				// If no token found, no further work possible
+				if (string.IsNullOrEmpty(token))
+				{
+					return AuthenticateResult.NoResult();
+				}
 
-			SecurityToken validatedToken;
-			var validators = Options.SecurityTokenValidatorsFactory();
-			foreach (var validator in validators)
-			{
-				if (validator.CanReadToken(token))
+				var validationParameters = Options.TokenValidationParameters.Clone();
```

Comment:
What is the meaning of this line?

Reply:
This line already exist, I refactor this change to be more clear. Here are many changes but are for indentation changes